lib/ohloh_scm/adapters/svn/misc.rb

```diff
@@ -123,6 +123,15 @@ def export(dest_dir, commit_id = final_token || 'HEAD')
 			run "svn export --trust-server-cert --non-interactive --ignore-externals --force -r #{commit_id} '#{SvnAdapter.uri_encode(File.join(root, branch_name.to_s))}' '#{dest_dir}'"
 		end
 
+    def base_path
+      url.sub(/(.*)(branches|trunk|tags)(.*)/, '\1').chomp('/')
+    end
+
+    def export_tag(dest_dir, tag_name)
+      tag_url = "#{base_path}/tags/#{tag_name}"
+			run "svn export --trust-server-cert --non-interactive --ignore-externals --force '#{tag_url}' '#{dest_dir}'"
```

Comment:
Please fix this indentation.
_You can select the line in `vim` and run `:retab`_